modules/aggregation-layers/src/utils/gpu-grid-aggregation/gpu-grid-aggregator.js

```diff
@@ -515,7 +515,7 @@ export default class GPUGridAggregator {
         });
       }
       framebuffers[id].resize(framebufferSize);
-      equations[id] = EQUATION_MAP[operation];
```

Comment:
Should this not fail? Silently changing the the requested operation seems like it could lead to tricky bugs (though I'll admit I'm not totally clear on where `operation` is coming from and what's expected of it).

Reply:
+1

Reply:
This is documented in the layer's page, `aggregation` prop, when not specified on invalid, we will fallback to `SUM`.